osc-dm-product-srv/src/utl/tag_generator.py

```diff
@@ -0,0 +1,28 @@
+from utl.openaillm import OpenAIClient
+import yaml
+import logging
+
+logger = logging.getLogger(__name__)
+def _create_tags(data, available_tags, vendor=None, model="gpt-4o",):
+    if model is None:
+        model = "gpt-4o"
```

Comment:
set default model in some file somewhere - I usually prefer something like <root_package>.common.constants.py - and pull default values from there. Also, is this default not made redundant by 4o already being the default in method signature?